.github/workflows/run-tests.yml

```diff
@@ -1,4 +1,4 @@
-# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# This workflow will install Python dependencies, run tests, lint, and perform static code analysis
```

Comment:
I think possibly here and in the wiki we need to add a comment about the fact we are testing a docker image. Are we just testing an image will launch? Or what aspect of the code is being tested by the Docker unittest. That sort of thing :) I think we also need a comment in the Docker.unittest file. I've just had a quick look and is it correct that the Docker image is launched and then unitests are run withing the image?

Reply:
Yes that's correct, we build an image to run the unit tests in. This image is created from the same `requirements.txt` as the production image but is not the same image